### PR TITLE
Issue 1

### PR DESCRIPTION
**Pull request by JihengNie**
image
image
image

### CODE REVIEW

**Comment by JihengNie** [comment]
image

**Comment by JihengNie** [comment]
image
image
image

File: index.html [hunk 1]
```diff
@@ -16,11 +16,44 @@
...
 </head>
 <body>
-  <header>
-  </header>
   <main>
+    <div class="heading">
+      <header class="container">
+        Code Journal
+      </header>
+    </div>
```
Comment (codingcodymiller) [inline index.html]: You should apply the background color to `header`, and give that element a child with the `container` class. Within that container, make a `row` and give that `row` a child with a `column-` class, then us an h1-h6 element for this text. That should vastly reduce the amount of special styling you need to apply to the `heading` class here.

File: index.html [hunk 2]
```diff
@@ -16,11 +16,44 @@
...
+    <div data-view="entry-form">
+      <form class="container new-entry">
+        <div class="row">
+          <h1>
+            New Entry
+          </h1>
```
Comment (codingcodymiller) [inline index.html]: No element should be a direct child of a `row` without a `column-` class.

File: index.html [hunk 3]
```diff
@@ -16,11 +16,44 @@
...
+            <label for="photoUrl">
+              <input required type="text" class="text-adjust" id="photoUrl">
+            </label>
+          </div>
+        </div>
+        <div class="row">
+          <h3 class="column-full"> Notes </h3>
+          <label for="comment"></label>
+          <textarea required class="text-adjust" id="comment" rows="10"></textarea>
+          <input class="save-button" type="submit" value="SAVE"></input>
```
Comment (codingcodymiller) [inline index.html]: You should have all of this in a `column-full` element.

File: js/main.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+var $formElement = document.querySelector('form');
+$formElement.addEventListener('submit', formSubmitted);
+
+function formSubmitted(event) {
+  event.preventDefault();
+  var newEntry = {
+    title: $textInputs[0].value,
+    photoUrl: $textInputs[1].value,
+    comment: $textInputs[2].value,
```
Comment (codingcodymiller) [inline js/main.js:19]: Use the named properties rather than index numbers, it is far less prone to bugs and misunderstandings.

File: js/main.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    EntryId: data.nextEntryId
```
Comment (codingcodymiller) [inline js/main.js]: Property names should not begin with a capital letter, you should name this `entryId`

File: js/main.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+  $photo.src = 'images/placeholder-image-square.jpg';
+  $formElement.reset();
+}
+
+window.addEventListener('beforeunload', storingData);
+
+function storingData(event) {
+  var dataJSON = JSON.stringify(data);
+  localStorage.setItem('code-journal', dataJSON);
+}
```
Comment (codingcodymiller) [inline js/main.js]: Please move this to `data.js` so that it is in the same file as the variable it is preserving.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**